README.md

```diff
@@ -155,6 +156,7 @@
 - [Github Profile README Generator](https://github.com/rahuldkjain/github-profile-readme-generator) - This tool provides an easy way to create github profile readme with latest addons like `visitors count`, `github stats` etc.
 - [Dynamic Profile Page On Github](https://github.com/umutphp/github-action-dynamic-profile-page) - Get dynamically generated list of your commits (of the repositories that the action is configured) on GitHub profile readme.
 - [npm package downloads](https://github.com/maddhruv/github-readme-npm-downloads) - Show all of your npm packages and their total downloads
+- [Markdown Badges](https://github.com/Ileriayo/markdown-badges)
```

Comment:
Provide description

Reply:
An opensource repo for tech badges suitable for markdown files.

Reply:
I mean provide description in the README